unittests/runtime/Stdlib.cpp

```diff
@@ -155,3 +157,81 @@ SWIFT_CC(swift) SWIFT_RUNTIME_STDLIB_INTERNAL
 void *$Ss32_getErrorEmbeddedNSErrorIndirectyyXlSgSPyxGs0B0RzlF(void *) {
   abort();
 }
+
+// Hashable
+
+SWIFT_RUNTIME_STDLIB_INTERNAL
+const char $Ss8HashableMp[1] = {0};
+
+// Array
+
+SWIFT_RUNTIME_STDLIB_INTERNAL
+const char $SSaMn[1] = {0};
+
+// Dictionary
+
+SWIFT_RUNTIME_STDLIB_INTERNAL
+const char $Ss10DictionaryVMn[1] = {0};
+
+// Set
+
+SWIFT_RUNTIME_STDLIB_INTERNAL
+const char $Ss3SetVMn[1] = {0};
+
+// Mirror
```

Comment:
I think that we should go ahead and use a macro here

```
// stub symbols for the protocol witness table and the type metadata accessor
#define STUB_TYPE_INFO(type)                                                   \
  SWIFT_RUNTIME_STDLIB_INTERNAL const char $Ss12_##type##VMa[1] = {0};         \
  SWIFT_RUNTIME_STDLIB_INTERNAL const char $Ss12_##type##Vs01_B0sWP[1] = {0};


STUB_TYPE_INFO(ClassMirror)
STUB_TYPE_INFO(EnumMirror)
STUB_TYPE_INFO(MetatypeMirror)
STUB_TYPE_INFO(OpaqueMirror)
STUB_TYPE_INFO(StructMirror)
STUB_TYPE_INFO(TupleMirror)

```

Reply:
`12` is the identifier length, so it can't be buried in the macro unfortunately. We used to have a macro for this when we were staging in the new mangling, but it looks like it's been taken out since.